packages/node/src/login/oidc/ClientRegistrar.spec.ts

```diff
@@ -24,8 +24,8 @@ import { mockStorageUtility } from "@inrupt/solid-client-authn-core";
 import ClientRegistrar from "./ClientRegistrar";
 import {
   IssuerConfigFetcherFetchConfigResponse,
-  mockDefaultIssuerConfig,
-  mockIssuerConfig,
+  mockDefaultIssuerMetadata,
```

Comment:
I'm just curious - what's the ratonale behind the name change from `Config` to `Metadata`? (This is referring to the JSON returned from the `.well-known/openid-configuration` endpoint, right (in which case I would have thought `Config` was more appropriate)...?)

Reply:
It's because internally, we have a `IIssuerConfig` class, which we use in our interfaces, and there exists a similar class from `openid-client`, `IssuerMetadata`. I started mixing them up, so I used a clearer name for the mocks :)

Reply:
Ah. Ok, well, in that case (i.e. if it confused me, and even you were getting confused!), then I'd try to make it even clearer again in the mocks, i.e.
```suggestion
  mockDefaultOpenIdClientIssuerMetadata,
```
...although that doesn't work so well either, mainly 'cos 'openid-client' is such a terribly unwieldy name (much like Solid Client Library is an unwieldy name, and why something like LIT works so much better instead... oh well...).
So whatever you think yourself - maybe it just looks strange here side-by-side with the our IIssuerConfig in this PR's diff, but if we can't come up with a better name that distinguishes our IssuerConfig and openid-client's one, then maybe just a comment on the mock definition, or something.